.travis.yml

```diff
@@ -25,5 +25,27 @@ jobs:
       script: make test BUILDTAGS=""
 
     - stage: validate
+#POWERON SUPPORT 
+    - stage: build
+      script: make
+      arch: ppc64le
+    - stage: lint
+      script: make lint BUILDTAGS=""
+      arch: ppc64le
+    - stage: test
+      script: make test BUILDTAGS=""
+      arch: ppc64le
+      go:  1.14.x
+    - stage: test
+      script: make test BUILDTAGS=""
+      arch: ppc64le
+      go: 1.15.x
+    - stage: test
+      script: make test BUILDTAGS=""
+      arch: ppc64le
+      go : tip
+    - stage: validate
+      arch: ppc64le
```

Comment:
I can think we can simplify it more.

The go versions are already specified at the top and Travis will construct the matrix on it's own. So we don't need to specify them here.

In fact, we don't need to add new stages at all but extend the existing ones with:
```
arch:
  - amd64
  - ppc64le
```

Reply:
@vrothberg  , I have tried earlier and it did not work , This was the reason i have specified cpu arch under matrix .

Again i tried as you suggested and ran the jobs and only new arch included in the test stage , I am expecting it should be  included in other stages Validate,build ,lint etc.. , Please find details below 


https://travis-ci.com/github/asellappen/selinux/builds/190412878

Please let me if you have any other thoughts.

Reply:
shall go for the approach which i used in matrix?